src/pages/ShowResource.tsx

```diff
@@ -17,24 +16,37 @@ const getComitResourceFn = async ({ resourcePath }: any) => {
 function ShowResource({ location }: RouteComponentProps) {
   const resourcePath = location.pathname.replace("/show_resource/", "");
 
-  const { data, isLoading, error } = useAsync({
-    promiseFn: getComitResourceFn,
-    resourcePath,
-    watch: location.pathname
```

Comment:
:x: This is still necessary in order to fetch the right resource when the path of the url changes.